src/ComponentBase.ts

```diff
@@ -44,7 +44,13 @@ interface StoreSubscriptionInternal<P, S> extends StoreSubscription<P, S> {
     _subscriptionKey?: string;
 }
 
-export abstract class ComponentBase<P extends {}, S extends Dictionary<any>> extends React.Component<P, S> {
+interface InternalState {
+    _getInstance: () => ComponentBase<any, any>;
```

Comment:
I'd prefer to name this _resub_getInstance or something to indicate that it's the library doing this